src/Jackett.Common/Indexers/FunFile.cs

```diff
@@ -77,20 +76,25 @@ public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken
         protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
         {
             var releases = new List<ReleaseInfo>();
+
             var qc = new NameValueCollection
             {
                 {"incldead", "1"},
                 {"showspam", "1"},
                 {"cat", MapTorznabCapsToTrackers(query).FirstIfSingleOrDefault("0")}
             };
-            if (!string.IsNullOrWhiteSpace(query.GetQueryString()))
+            if (query.IsImdbQuery)
+            {
+                qc.Add("search", query.ImdbID);
+                qc.Add("s_desc", "1");
+            }
+            else if (!string.IsNullOrWhiteSpace(query.GetQueryString()))
                 qc.Add("search", query.GetQueryString());
```

Comment:
I've been checking in trackers to see if they accept an empty query string and skipping the `IsNullOrWhiteSpace` check if they do. Can you test on the site to see if `&search=&` in your search url returns the expected results and update accordingly?

Other than that, this passes muster and this isn't a blocking change.

Reply:
Changed. It works.